src/sections/Brand/Brand-components/meshery-operator.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styled from "styled-components";
+import { Row, Col } from "../../../reusecore/Layout";
+import Button from "../../../reusecore/Button";
+import ColorBox from "../../../components/ColorBox";
+import { FiDownloadCloud } from "react-icons/fi";
+import MesheryOperatorDark from "../../../assets/images/meshery-operator/meshery-operator-dark.png";
+import MesheryOperator from "../../../assets/images/meshery-operator/meshery-operator.png";
+
+const MesheryOperatorWrapper = styled.section`
+    .sub-project {
+        margin-top: -2rem;
+    }
+
+    .mesheryOperator{
+        background: ${props => props.theme.primaryColor};
+    }
+`;
+
+const MesheryOperatorBrand = () => {
+    return (
+        <MesheryOperatorWrapper>
+            <div className="post-content">
+                <Row className="sub-project brand-section">
+                    <Col xs={12} sm={6}>
+                        <h2 className="layerH3">
+                            Meshery Operator
+                        </h2>
+                    </Col>
+                    <Col xs={12} sm={6} className="download-button">
+                        <Button primary title="Donwload Logo Assets" url="../../../assets/brand/meshery-operator-brand-kit.zip" external={true}>
+                            <FiDownloadCloud size={21} className="icon-left" />
+                        </Button>
+                    </Col>
+                    <Col>
+                        <p>
+                            The Meshery Operator mark includes the Meshery Operator name & logo, and any word,
+                            phrase, image, or other designation that identifies the source or
+                            origin of any Layer5 projects. Please don’t modify the marks or
+                            use them in a confusing way, including suggesting sponsorship or
+                            endorsement by Layer5, or in a way that confuses Layer5 with
+                            another brand (including your own).
+                        </p>
+                    </Col>
+                </Row>
+                <Row>
+                    <Col>
+                        <h3 className="layerH3 in">
+                            Logos
+                        </h3>
+                    </Col>
+                    <Row className="ImgDiv">
+                        <Col xs={12} sm={4}>
+                            <img src={MesheryOperatorDark} />
```

Comment:
Can you add `alt` here and any other place missing

Reply:
Sure.